docs/contrib/bindgen.rst

```diff
@@ -0,0 +1,40 @@
+Automatic binding generation
+============================
+
+Binding generation for now is experimental. The goal is to first use it
+internally to generate bindings for C's stdlib and later make it
+available via scala.meta macros and sbt tasks.
```

Comment:
IDE support is a blocker here, if scala.meta macros don't work well enough we might have to use code generation instead.